accounts/models.py

```diff
@@ -3,19 +3,56 @@
 from django.contrib.auth import get_user_model
 from django.contrib.auth.models import AbstractUser
 from django.db import models
+from phonenumber_field.modelfields import PhoneNumberField
 
 
 class User(AbstractUser):
     # implicit username, email, first_name, and last_name fields
     # from AbstractUser that contains the user's PennKey
     pennid = models.IntegerField(primary_key=True)
     uuid = models.UUIDField(unique=True, default=uuid.uuid4, editable=False)
+    preferred_name = models.CharField(max_length=225, blank=True)
+
+    VERIFICATION_EXPIRATION_MINUTES = 10
+
+    def get_preferred_name(self):
+        if self.preferred_name != "":
+            return self.preferred_name
+        else:
+            return self.first_name
 
 
 class Student(models.Model):
     user = models.OneToOneField(get_user_model(), on_delete=models.DO_NOTHING)
     major = models.CharField(max_length=255, blank=True)
```

Comment:
We still need to convert major and school into separate models, right? @JayVish

Reply:
Should I remove major & school from the Student model then? @ArmaanT

Reply:
@ArmaanT Sorry, I hadn't seen this before. Yeah, I've converted majors into a separate major model on the add-major-school branch. I am working on doing the same for the school model as well.

Reply:
So if we map from the major & school model to the student model via one-to-one field, I can just remove major & school fields from the student model side?

Reply:
I think what we should do is wait for @JayVish to finish up the school/major features on his branch, then merge those into this one and then merge this into master